src/main/java/org/cryptomator/frontend/fuse/ReadWriteAdapter.java

```diff
@@ -67,7 +67,8 @@ protected int checkAccess(Path path, Set<AccessMode> requiredAccessModes) {
 	public int mkdir(String path, @mode_t long mode) {
 		try (PathLock pathLock = lockManager.createPathLock(path).forWriting();
 			 DataLock dataLock = pathLock.lockDataForWriting()) {
-			Path node = resolvePath(path);
+			var nioSuitablePath = fileNameTranscoder.fuseToNio(path);
+			Path node = resolvePath(nioSuitablePath);
```

Comment:
I'd prefer (at least as long as we don't nest a lot more calls)

```suggestion
			Path node = resolvePath(fileNameTranscoder.fuseToNio(path));
```

Same applies for the remaining changes in this class.

Reply:
Fixed in 04b0368